public/locales/fr/countries.json

```diff
@@ -0,0 +1,224 @@
+ {   
+	 "entries": {	 
```

Comment:
is there no library for country listing with translation?

Reply:
I found a lib for country listing with translation https://github.com/michaelwittig/node-i18n-iso-countries
For os/browser emoticon you'r right i didn't think about it. It will be better. I'll implement it

Reply:
i tried with browser emoticon it doesn't work. I'll try third party lib like https://github.com/lipis/flag-icon-css